### PR TITLE
Implement Footer-Component

### PR DESCRIPTION
**Pull request by mskradde**
Adding my Footer to my App
#24

### CODE REVIEW

**Review by lmachens — COMMENTED** [review]
Good job removing the old files (should be in a different pull request next time ;) ).
Are you sure that you added all changes to the footer component? It looks like something is missing.

**Comment by mskradde** [comment]
> Good job removing the old files (should be in a different pull request next time ;) ).
> Are you sure that you added all changes to the footer component? It looks like something is missing.

Now you should see all changes :)

File: client/src/components/Footer.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+import AccountIcon from "../stories/assets/account-icon.svg";
+import ChatIcon from "../stories/assets/message-icon.svg";
+import SettingsIcon from "../stories/assets/settings-icon.svg";
+
+const MainFooter = styled.footer`
+  width: 100%;
+
+  background: #ade1cd;
+  border: none;
+  border-radius: 0;
```
Comment (lmachens) [inline client/src/components/Footer.js]: I think these props are not required for a footer. A footer shouldn't have a border by default.

File: client/src/components/Footer.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+const Navbar = styled.nav`
+  display: flex;
+  margin: 0.2 em;
+  justify-content: space-around;
+  align-items: center;
+`;
+
+const NavImage = styled.img`
+  margin: 10px 0;
+  opacity: ${({ activeTab }) => (activeTab ? "0.5" : "1")};
```
Comment (lmachens) [inline client/src/components/Footer.js]: `activeTab` doesn't feel like a good name, because it's not a tab.
I would suggest `active`.

**Review by lmachens — COMMENTED** [review]
Good job 🎉 ! Only small comments.

**Pull request merged**